packages/client/src/components/GrantNotes.vue

```diff
@@ -50,13 +50,19 @@
       </b-form-group>
     </div>
 
-    <!-- Current User's Note -->
-    <GrantNote
+    <!-- Users Note -->
+    <UserActivityItem
       v-if="userNote && !editingNote"
-      data-test-user-note
       :class="userNoteClass"
-      :note="userNote"
+      :user-name="userNote.user.name"
+      :user-email="userNote.user.email"
+      :team-name="userNote.user.team.name"
+      :avatar-color="userNote.user.avatarColor"
+      :created-at="userNote.createdAt"
```

Comment:
This block appears to be missing something like `:is-edited="userNote.isRevised"` – when testing in my local environment, I'm missing the `(edited)` part of the `{{ timeElapsedString }} (edited)` part of the [template](https://github.com/usdigitalresponse/usdr-gost/blob/20dc496537347859d82798c9e65eacfb7e990db5/packages/client/src/components/UserActivityItem.vue#L36-L42).

Reply:
thanks for catching, I added this back in